pages/[report]/[fight]/index.tsx

```diff
@@ -81,6 +88,41 @@ const Fight = (fightResponse: IFightResponse) => {
       endTime: query.endTime || '',
     };
     fetchFightData({ setFightData, params });
+  };
+
+  const fetchParseDPSData = async () => {
+    const { data } = await fetchFightParseData({
+      action: 'LIST_PARSE_TO_FIGHT',
+      code: query?.report || '',
+      encounterID: query.encounterID || '',
+      parseType: 'dps',
+    });
+    setParseData((prevState) => ({
+      ...prevState,
+      dps: data.dps,
+    }));
```

Comment:
couldn't you do something like: 
```suggestion
    setParseData((prevState) => ({
      ...prevState,
      ...data,
    }));
```
then you also wouldn't need `fetchParseDPSData `and `fetchParseHPSData `separately, as you could create a parametrized function `fetchParseData = async (parseType: ParseType) => ... ` and call that twice in L123